config/backend_manager.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"sync"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/ory/fosite"
+	"github.com/ory/hydra/client"
+	"github.com/ory/hydra/consent"
+	"github.com/ory/hydra/jwk"
+	"github.com/ory/hydra/pkg"
+)
+
+var (
+	backends = make(map[string]BackendManager)
+	bmutex   sync.Mutex
+)
+
+type BackendManager interface {
```

Comment:
I think `Connection` or `Connector` would be a better choice here :)

Reply:
Sure, so `BackendConnector` then? Any thoughts on what variable we should load from the database plugins? I'm using `Manager` right now but can update this to `Connector` to match this name change.

Reply:
Yup, `BackendConnector` or `Connector` both work fine. Choose one you like. The plugin should export the same name, so also either `BackendConnector` or `Connector`.